app/views/failed_registration_mailer/notify.html.erb

```diff
@@ -0,0 +1,4 @@
+<h1>Retirement Adviser Directory Registration</h1>
+<div>
+  We haven't been able to verify your Firm Reference Number against the FCA register. Please contact us at radenquiries@moneyadviceservice.org.uk and we will help you with your registration.
```

Comment:
If we setup the env var we can use and replace "radenquiries@moneyadviceservice.org.uk". An idea...

Reply:
Please refer to the previous comment about the env var.